app/src/js/components/Header/header.js

```diff
@@ -17,7 +17,7 @@ import { kibanaAllLogsLink } from '../../utils/kibana';
 import { getPersistentQueryParams } from '../../utils/url-helper';
 
 const paths = [
-  [strings.collections, '/collections'],
+  [strings.collections, '/collections/all'],
```

Comment:
Changed to /collections/all because /collections redirects to /collections/all and all query params were stored for /collections/all, but for s to /collections params were not appended.